utensor_cgen/core.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, pb_file, idx_dir, embed_data_dir, method, debug_cmt=False, ou
     self.method = method
     self.debug_cmt = debug_cmt
     self.output_nodes = output_nodes
+    self.quantize_enabled = quantized_flag
```

Comment:
In `generate`, you forget to pass `self.quantize_enabled` flag to `parse_pb` function call.

Reply:
Fixed